LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -145,6 +145,41 @@ function dispatchTouchActions(actions, options = { insertPauseAfterPointerUp: fa
     })();`, resolve));
 }
 
+async function dispatchWheelActions(actions)
+{
+    if (!window.eventSender)
+        throw new Error("window.eventSender is undefined.");
+
+    for (let action of actions) {
+        if (action.type == "pause") {
+            await pause(action.duration);
+            continue;
+        }
+
+        if (action.type != "scroll")
+            throw new Error(`Unrecognized wheel action type: ${action.type}`);
+
+        let { x, y, origin, deltaX, deltaY } = action;
+        if (origin instanceof Element) {
+            const bounds = origin.getBoundingClientRect();
+            logDebug(() => `${origin.id} [${bounds.left}, ${bounds.top}, ${bounds.width}, ${bounds.height}]`);
+            x += bounds.left + 1;
+            y += bounds.top + 1;
+        }
+        eventSender.monitorWheelEvents();
```

Comment:
I'd prefer this use the more modern `uiController.sendEventStream()`; see `UIHelper.mouseWheelSequence()`.

Reply:
Using this causes tests to hang that passed using `mouseScrollByWithWheelAndMomentumPhases`. When using `await uiController.sendEventStream()`, I saw the events were not even getting to the UIScriptController, but without awaiting they were getting though. My assumption is that it may be necessary to call `await uiController.sendEventStream()` on another thread, which the tests are not doing. I think it makes sense to stick with the previous method for now.